### PR TITLE
852 details page test page object

### PR DESCRIPTION
**Pull request by norisalsaadie**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,8 +32,8 @@ start_dev: ## start with dev build image, with reviewer-mocks mocking continuum
 	${DOCKER_COMPOSE} pull reviewer-mocks nginx
 	$(MAKE) build_dev
 	${DOCKER_COMPOSE} up -d reviewer-client reviewer-mocks
-	./.scripts/docker/wait-healthy.sh reviewer-client_app 210
```
Comment (will-byrne) [inline Makefile:35]: Do we not need to wait for them to be up anymore?

File: Makefile [hunk 2]
```diff
@@ -32,8 +32,8 @@ start_dev: ## start with dev build image, with reviewer-mocks mocking continuum
...
-	./.scripts/docker/wait-healthy.sh reviewer-client_mocks 20
+	# ./.scripts/docker/wait-healthy.sh reviewer-client_app 210
```
Comment (erezmus) [inline Makefile:35]: was is this commented?

Comment (norisalsaadie) [reply Makefile:35]: Opps

File: Makefile (hunk 1 above)
Comment (norisalsaadie) [reply Makefile:35]: accidental commit

**Pull request merged**